libs/expo/betterangels/src/lib/static/constants.ts

```diff
@@ -6,13 +6,20 @@ export const ServicesByCategory = [
     items: [
       ServiceEnum.Food,
       ServiceEnum.Water,
+      ServiceEnum.Bag,
+      ServiceEnum.Batteries,
       ServiceEnum.Blanket,
       ServiceEnum.Book,
       ServiceEnum.Clothes,
+      ServiceEnum.ClothesSocks,
```

Comment:
How did this item make it in again? We mentioned that we were going to collapse ALL sections into general categories for now.